source/numbers.rst

```diff
@@ -42,6 +42,7 @@ You just type them as literals::
 Questions
 ---------
```

Comment:
@maxspencer I am pretty sure the computers we are using for the course won't have Python 3 installed. It is worth asking if we can change that.

It may be safer to plan for Python 2... Lets chat more about it today.